angelssmelting/prototypes/buildings/induction-furnace.lua

```diff
@@ -232,9 +232,9 @@ data:extend(
           pipe_covers = pipecoverspictures(),
           base_level = 1,
           pipe_connections = {{ position = {2, -3} }}
-        }
+        },
+        off_when_no_fluid_recipe = true,
       },
```

Comment:
Yes, we should not set this, but I do want to keep the value in the correct place
```suggestion
        },
        --off_when_no_fluid_recipe = true,
      },
```
or
```suggestion
        },
        off_when_no_fluid_recipe = false,
      },
```

Reply:
I went for the commented variety